### PR TITLE
Add context menu to Show All Character toolbar button

### PR DESCRIPTION
**Pull request by ozone10**
Improve situation for #14832
Make it easier to toggle various non-printing symbols.

### CODE REVIEW

**Comment by ozone10** [comment]
image

**Comment by donho** [comment]
@ozone10 
Nice PR.
However, "Show Indent Guide" & "Show Wrap Symbol" should not be in the context menu IMO.
Is it difficult to remove them technically?

**Comment by ozone10** [comment]
@donho 
it is complicated, I would need to create new menu from existing one and that is quite hard.

**Comment by ArkadiuszMichalski** [comment]
Something similar was already proposed, but unfortunately rejected https://github.com/notepad-plus-plus/notepad-plus-plus/pull/9765. Maybe the previous code will be useful so I leave a reference.

**Comment by ozone10** [comment]
@donho 
@ArkadiuszMichalski's reference gave me idea how to modify dropdown menu.

image

**Comment by donho** [comment]
> Improve situation for #14832

@ozone10 
Can we consider #14832 is fixed by the PR?
BTW, if you could add a separator between the 2 icons "Show indent guide" & "Define your langues", it would be great.
Otherwise, I will add it.

**Comment by ozone10** [comment]
@donho 
It is workaround for what is requested, so it could be closed.

btw I would also add separator between  Folder as workspace and Monitoring, or move Monitoring to next to Synchronize Scrolling buttons

**Comment by donho** [comment]
> I would also add separator between Folder as workspace and Monitoring

OK

>  move Monitoring to next to Synchronize Scrolling buttons

I don't feel this way.

File: PowerEditor/src/Notepad_plus.cpp [hunk 1]
```diff
@@ -99,6 +99,11 @@ ToolBarButtonUnit toolBarIcons[] = {
...
+
+    //---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------//
+    {0,                                IDI_SEPARATOR_ICON,         IDI_SEPARATOR_ICON,            IDI_SEPARATOR_ICON,         IDI_SEPARATOR_ICON,            IDI_SEPARATOR_ICON,            IDI_SEPARATOR_ICON,               IDI_SEPARATOR_ICON,            IDI_SEPARATOR_ICON,            IDI_SEPARATOR_ICON},
+    //---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------//
+
```
Comment (donho) [inline PowerEditor/src/Notepad_plus.cpp:106]: @ozone10 
I thought you want to add also separator between Folder as workspace and Monitoring ?

Comment (ozone10) [reply PowerEditor/src/Notepad_plus.cpp:106]: sorry I was away

**Comment by u07** [comment]
1) Guys, thanks for the shortcut. It'll make it easier.
2) Is there a nightie build to test?  If not, can you do the next, please:

- open NPP
- click the “¶” to disable specials
- drag a binary to the NPP window

See any specials? I still do:

image

Now switch to another tab and back. Now it looks like this:

image

**Pull request closed without merging**